csharp/private/frameworks/net20.BUILD

```diff
@@ -0,0 +1,668 @@
+        "//:System.Drawing",
+        "//:System.Management",
+        "//:System.Runtime.Remoting",
+        "//:System.Web",
+        "//:System.Windows.Forms",
+    ],
+    visibility = ["@net//:__pkg__"],
+)
+
+import_library(
+    name = "Microsoft.VisualBasic.Compatibility",
+    target_framework = "net20",
+    refdll = "//:build/.NETFramework/v2.0/Microsoft.VisualBasic.Compatibility.dll",
+    deps = [
+        "//:Microsoft.VisualBasic",
+        "//:mscorlib",
+        "//:System",
+        "//:System.Design",
+        "//:System.Drawing",
+        "//:System.Web",
+        "//:System.Windows.Forms",
+    ],
+    visibility = ["@net//:__pkg__"],
+)
+
+import_library(
+    name = "Microsoft.VisualBasic.Compatibility.Data",
+    target_framework = "net20",
+    refdll = "//:build/.NETFramework/v2.0/Microsoft.VisualBasic.Compatibility.Data.dll",
+    deps = [
+        "//:Microsoft.VisualBasic",
+        "//:Microsoft.VisualBasic.Compatibility",
+        "//:mscorlib",
+        "//:System",
+        "//:System.Drawing",
+        "//:System.Security",
+        "//:System.Windows.Forms",
+    ],
+    visibility = ["@net//:__pkg__"],
+)
+
+import_library(
+    name = "Microsoft.VisualBasic.Vsa",
+    target_framework = "net20",
+    refdll = "//:build/.NETFramework/v2.0/Microsoft.VisualBasic.Vsa.dll",
+    deps = [
+        "//:Microsoft.Vsa",
+        "//:Microsoft_VsaVb",
+        "//:mscorlib",
+        "//:System",
+    ],
+    visibility = ["@net//:__pkg__"],
+)
+
+import_library(
+    name = "Microsoft.VisualC",
+    target_framework = "net20",
+    refdll = "//:build/.NETFramework/v2.0/Microsoft.VisualC.dll",
+    deps = ["//:mscorlib"],
+    visibility = ["@net//:__pkg__"],
+)
+
+import_library(
+    name = "Microsoft.Vsa",
+    target_framework = "net20",
+    refdll = "//:build/.NETFramework/v2.0/Microsoft.Vsa.dll",
+    deps = [
+        "//:mscorlib",
+        "//:System",
+    ],
+    visibility = ["@net//:__pkg__"],
+)
+
+import_library(
+    name = "Microsoft.Vsa.Vb.CodeDOMProcessor",
+    target_framework = "net20",
+    refdll = "//:build/.NETFramework/v2.0/Microsoft.Vsa.Vb.CodeDOMProcessor.dll",
+    deps = [
+        "//:mscorlib",
+        "//:System",
+        "//:System.Xml",
+    ],
+    visibility = ["@net//:__pkg__"],
+)
+
+import_library(
+    name = "Microsoft_VsaVb",
+    target_framework = "net20",
+    refdll = "//:build/.NETFramework/v2.0/Microsoft_VsaVb.dll",
+    deps = [
+        "//:Microsoft.Vsa",
+        "//:mscorlib",
+    ],
+    visibility = ["@net//:__pkg__"],
+)
+
+import_library(
+    name = "mscorlib",
+    target_framework = "net20",
+    refdll = "//:build/.NETFramework/v2.0/mscorlib.dll",
+    visibility = ["@net//:__pkg__"],
+)
+
+import_library(
+    name = "sysglobl",
+    target_framework = "net20",
+    refdll = "//:build/.NETFramework/v2.0/sysglobl.dll",
+    deps = [
+        "//:mscorlib",
+        "//:System.Xml",
+    ],
+    visibility = ["@net//:__pkg__"],
+)
+
+alias(
```

Comment:
@jimevans so this is how we're dealing with cycles: When you reference `//:System` you're actually referencing `//:System.Xml`. That's OK because `//:System.Xml` has a reference to `//:System-noncyclic` (the next rule below). That rule doesn't have a dep on `//:System` because Bazel (reasonably) won't allow cycles.